content/news/lottie-1.0-draft-review/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Lottie Animation Community announces public review draft of Lottie v1.0 specification
+date: "2024-08-01T12:01:01.284Z"
+description: "The Lottie Animation Community (LAC), a Joint Development Foundation project, is pleased to announce the public review draft of the v1.0 Lottie JSON specification."
+featuredImage: lac.png
+---
+
+### Lottie Animation Community announces public review draft of Lottie v1.0 specification
+
+The Lottie Animation Community (LAC), a Joint Development Foundation project, is pleased to announce the public review draft of the v1.0 Lottie JSON specification. v1.0 will be the first major step in securing the [Lottie File Format](https://lottie.github.io/) as an open standard for efficient, scalable and cross-platform animated vector graphics.
+
+The public review period for the Lottie v1.0 draft will be from July 29 to August 26, 2024. We request all significant changes be proposed by Aug 19 to allow for final review and publishing. Review the documentation [here](https://lottie.github.io/lottie-spec/) and validate your existing Lotties [here](https://lottie.github.io/lottie-spec/validator). Participate in the review by opening github issues for the spec [here](https://github.com/lottie/lottie-spec/issues).
```

Comment:
Are the dates confirmed?

Reply:
The date is flexible, we can fill it in once we get everything else ready.